lore/settings.py

```diff
@@ -237,7 +237,9 @@ def get_var(name, default):
 if LORE_USE_S3:
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 else:
-    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+    # by default use django.core.files.storage.FileSystemStorage with
+    # overwrite feature
+    DEFAULT_FILE_STORAGE = 'storages.backends.overwrite.OverwriteStorage'
```

Comment:
This should fix the issue with tests, but do we need to make some changes to the S3 configuration as well?